src/commands/trigger-pipeline.yml

```diff
@@ -38,4 +38,4 @@ steps:
         PARAM_REPO_ENV_VAR: << parameters.repo-name >>
         PARAM_BRANCH_ENV_VAR: << parameters.branch >>
         CUSTOM_PARAMS: << parameters.custom-parameters >>
-      command: << include(../scripts/trigger-pipeline.sh) >>
+      command: \<< include(../scripts/trigger-pipeline.sh) >>
```

Comment:
```suggestion
      command: << include(../scripts/trigger-pipeline.sh) >>
```

Reply:
:+1: I've reverted those changes to keep the history clean

Reply:
thank you